common/app/common/pages/PageHeader.java

```diff
@@ -0,0 +1,31 @@
+package common.pages;
+
+import common.cms.CmsPage;
+
+public class PageHeader {
```

Comment:
PageHeader has nothing to do with CMS. PageHeader (as well as PageContent, PageFooter, etc.) is about the data that has to be injected in a template. This data can come from CMS, from SPHERE or from any other external service. Therefore it is not a wrapper, but rather a user of the CMS.